common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/ShapleyLoadOptions.java

```diff
@@ -0,0 +1,69 @@
+package ai.h2o.mojos.deploy.common.transform;
+
+/**
+ * Enum defining options for loading the mojo to enable Shapley predictions.
+ */
+public enum ShapleyLoadOptions {
+  ALL("ALL"),
+  NONE("NONE"),
+  ORIGINAL("ORIGINAL"),
+  TRANSFORMED("TRANSFORMED");
+
+  private final String value;
+
+  ShapleyLoadOptions(String value) {
+    this.value = value;
+  }
+
+  public String getValue() {
+    return value;
+  }
+
+  @Override
+  public String toString() {
+    return String.valueOf(value);
+  }
+
+  /**
+   * Checks whether Shapley scoring is permitted.
+   * @return {@link Boolean}
+   */
+  public boolean isEnabled() {
+    ShapleyLoadOptions options = ShapleyLoadOptions.fromValue(value);
+    switch (options) {
+      case ALL:
+      case ORIGINAL:
+      case TRANSFORMED:
+        return true;
+      case NONE:
+      default:
+        return false;
```

Comment:
I can see benefit of listing each case, just adding here to lend a fresh pair of eyes.  May be reducible to `return options != NONE`;